rules/autoconsent/inmobi.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "inmobi",
+    "prehideSelectors": [".accept-cookie"],
+    "detectCmp": [
+        {
+            "exists": ".accept-cookie > .accept-cookie-inner"
+        }
+    ],
+    "detectPopup": [
+        {
+            "visible": ".accept-cookie > .accept-cookie-inner"
+        }
+    ],
+    "optIn": [
+        {
+            "waitForThenClick": "xpath///*[contains(@class, 'accept-cookie')]//*[contains(text(), 'Accept')]"
```

Comment:
XPath selectors as the label attributes are incorrect a lot of the time. For example the Reject button often is labelled _Accept_. Site always seems to use English.